include/openssl/ssl.h

```diff
@@ -2170,7 +2178,7 @@ void SSL_CTX_set0_ctlog_store(SSL_CTX *ctx, CTLOG_STORE *logs);
  */
 const CTLOG_STORE *SSL_CTX_get0_ctlog_store(const SSL_CTX *ctx);
 
-# endif /* OPENSSL_NO_CT */
+# endif                         /* OPENSSL_NO_CT */
```

Comment:
Huh, this is the only instance in ssl.h of endif-with-comment.
I guess your change is consistent with ssl_locl.h, so I don't really have grounds to object.